docs/advanced_documentation/build-guide.md

```diff
@@ -466,3 +466,20 @@ the {{ "[`tests/package_tests`]({}/tests/package_tests)".format(gh_link_head_blo
 This project has the main project as a required dependency. Configuration will fail if the main project has not been
 built and installed, e.g. using `cmake --build --preset <preset> --target install` for the current preset.
 ```
+
+## Documentation
+
+The documentation is built in [Sphinx](https://github.com/sphinx-doc/sphinx). It can be built locally in python environment. The packages required for building it can be found under [docs]. The steps required for building the python package needs to be carried out first as per (#build-python-package). Then the documentation specific packages can be installed via:
```

Comment:
```suggestion
The documentation is built in [Sphinx](https://github.com/sphinx-doc/sphinx). It can be built locally in a Python environment. The packages required for building it can be found under the `[docs]` optional dependencies. In addition, the `power-grid-model` Python package needs to be built by following the steps mentioned in the [above](#build-python-package). After that, the documentation specific packages can be installed via:

```